cmd/mesh/shared.go

```diff
@@ -33,7 +33,9 @@ func initLogsOrExit(args *rootArgs) {
 
 func configLogs(logToStdErr bool) error {
 	opt := log.DefaultOptions()
-	if !logToStdErr {
+	if logToStdErr {
+		opt.OutputPaths = []string{"stderr"}
```

Comment:
does this work on windows?

Reply:
It should because it's the same target as the logger itself is using for stderr default output.